### ISSUE TITLE
Text not bold on Windows 7 VM - Download page

### ISSUE THREAD

**Issue by jwzimmer** [LLK/scratch-www#1437]
`/download` page, Staging, Chrome

On Chrome + Mac, the text below includes two bolded sections, "Note for Mac Users:" and "here". However, in Chrome + Windows 7 VM, the text is not bolded.

The text should be:
> **Note for Mac Users:** the latest version of Scratch 2.0 Offline requires Adobe Air 20. To upgrade to Adobe Air 20 manually, go **here**.

<img width="1132" alt="screen shot 2017-07-26 at 11 22 52 am" src="https://user-images.githubusercontent.com/1320456/28629853-9557e7de-71f6-11e7-8454-fc157582d352.png">

**Comment by TheGrits** [comment]
@jwzimmer Appears to have been already in production: 
![screenshot 2017-07-26 at 12 02 38](https://user-images.githubusercontent.com/12952370/28631180-5a9196fa-71fa-11e7-96b7-e64ee5737f38.png)

**Comment by jwzimmer** [comment]
@TheGrits Yeah, that is not surprising, thanks for checking. : ) This is not a high priority, just a little thing I noticed it might be nice to fix when there's time. 👍

**Comment by TheGrits** [comment]
That's strange, both the .jsx and the l10n.json files look perfectly fine:
- https://github.com/LLK/scratch-www/blob/develop/src/views/download/download.jsx#L91
- https://github.com/LLK/scratch-www/blob/develop/src/views/download/l10n.json#L4

**Comment by R4356th** [comment]
@benjiwheeler, this issue appears to have been resolved.
